src/utils/env.ts

```diff
@@ -97,3 +97,10 @@ export const getShowStablecoinAppUrl = () => {
     }
     return NEXT_PUBLIC_SHOW_STABLECOIN_APP_URL === 'true';
 };
+
+export const getReferralMessage = (): string => {
+    const referralMessage = process.env.NEXT_PUBLIC_REFERRAL_MESSAGE;
+
+    assert(referralMessage, 'NEXT_PUBLIC_REFERRAL_MESSAGE is not set');
```

Comment:
let's not throw an error here, if it's not a roadblocker to the application functionality.

Reply:
I fixed that!